### PR TITLE
Better help text for sync

### LINKED ISSUE #88: sync: Clearer message when no child environments found
```
$ vip sync --app 309
Could not find any child environments for ellaminnowpea-xyz
```

- Change "child" to "non-production"
- Prefix with "Error:"?
- Suffix with next steps: "To set up development environment, please contact VIP Support"

### PR DESCRIPTION
**Pull request by elazzabi**
This will displays a better help text when an environment do not have child environments

should we add an `Error:` 🤔 ?

### Test
1. pull the branch and switch to it
2. run `npm link`
3. run `vip sync --app=[ID]` where ID is the ID of an app without childs (try 636)

fixes #88

### CODE REVIEW

**Comment by joshbetz** [comment]
> should we add an Error: 🤔 ?

If you do, I would recommend trying `${ chalk.red( 'Error:' ) }` to see what it looks like.

I wonder if it's worth creating a helper function that prefixes errors with red 'Error' text.

File: src/lib/cli/command.js [hunk 1]
```diff
@@ -226,7 +226,8 @@ args.argv = async function( argv, cb ): Promise<any> {
 			options.env = env;
 		} else if ( ! options.app || ! options.app.environments || ! options.app.environments.length ) {
 			if ( _opts.childEnvContext ) {
-				console.log( `Could not find any child environments for ${ chalk.blueBright( options.app.name ) }` );
+				console.log( `Could not find any non-production environments for ${ chalk.blueBright( options.app.name ) }.` );
+				console.log( 'To set up a new development environment, please contact VIP Support.' );
 			} else {
 				console.log( `Could not find any environments for ${ chalk.blueBright( options.app.name ) }` );
```
Comment (mjangda) [inline src/lib/cli/command.js]: We should probably update the error here too?

Comment (elazzabi) [reply src/lib/cli/command.js]: when an env don't have child envs, this is the block that get executed. i'm not sure when the other block is executed 🤔

Comment (mjangda) [reply src/lib/cli/command.js]: Hm, yeah, I'm not sure how the condition is triggered :) I think we can probably drop the `if/else`.

Comment (elazzabi) [reply src/lib/cli/command.js]: yes, i think so. dropped here: https://github.com/Automattic/vip/pull/181/commits/fe650f743a4f6f59eb9a07cfa098796ebebbf2fb

**Review by mjangda — APPROVED** [review]
👍

**Pull request merged**